src/address_format.rs

```diff
@@ -0,0 +1,114 @@
+use super::*;
```

Comment:
What is the purpose of splitting this all up?

Reply:
To allow adding new code to the crate without creating one huge file that is hard to maintain.